Car.java

```diff
@@ -0,0 +1,55 @@
+/*
+Car class
+by Curtis Kunkle
+*/
+
+package CarPackage;
+
+public class Car {
+    
+    //fields
+    private int yearModel=0;
+    private String make = "";
+    private int speed;
+    
+    /**
+     * Constructor accepts yearModel and make. Sets speed to zero.
+     * @param yearModel: year model of car
+     * @param make: make of car
+     */
+    public Car(int yearModel, String make){
+        this.yearModel = yearModel;
+        this.make = make;
+        speed = 0;
+    }
+    
+    //Accessors
+    public String getMake(){return this.make;}
+    public int getYearModel(){return this.yearModel;}
+    public int getSpeed(){return this.speed;}
+    
+    /**
+     * Increases car's speed by 5
+     * Also outputs a message. This is unnecessary, but makes it 
+     * easier to understand on the console
```

Comment:
Interesting idea. In a production system we would condition these debugging statements with a debug flag.

That flag would be a constant at the beginning of your program that would instruct how your classes should behave. It can be a messy design though.